packages/docusaurus/src/client/theme-fallback/Root/index.tsx

```diff
@@ -16,5 +18,12 @@ import type {Props} from '@theme/Root';
 //
 // See https://github.com/facebook/docusaurus/issues/3919
 export default function Root({children}: Props): JSX.Element {
-  return <>{children}</>;
+  return (
+    <>
```

Comment:
Please don't add it here. There's currently no way to swizzle root with CLI, so as soon as the user creates an empty `Root.js` file (which is also recommended in our docs), this gets lost. Maybe add it in `App.tsx`?